docs/src/_guide/advanced-topics.md

```diff
@@ -14,6 +14,39 @@ Most of Meltano's features are available without installing any additional packa
 * `gcs` - Support for using Google Cloud Storage as a [state backend](/concepts/state_backends)
 * `azure` - Support for using Azure Blob Storage as a [state backend](/concepts/state_backends)
 
+## Extension Developer Kit (EDK)
+
+Meltano extensions are lightweight shims that allow you to integrate existing data tools with Meltano.
+Extensions allow the developer to add additional features like pre/post-hooks to run before/after Meltano executes the application, as well enabling project scaffolding to be customized for each plugin. This project scaffolding was previously accomplished via files bundles.
+
+The Meltano [Extension Developer Kit](https://github.com/meltano/edk) (EDK) was created to make it easier for developers to build Meltano extensions.
+For more information on how to build your own EDK-based Meltano extension, see [the EDK docs](https://edk.meltano.com/en/latest/).
+
+### EDK Context
```

Comment:
This section is more about how Meltano is classifying plugins and less about the EDK itself - as the EDK itself is agnostic to the plugin type we place upon it.


```suggestion
### Meltano Plugin Types
```

Or something like

```suggestion
### Meltano Utilities
```

Or

```suggestion
### Selecting a Plugin Type
```

Or

```suggestion
### Understanding Plugin Types
```